### PR TITLE
Introduce Pdf Viewer

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
As an example how file viewers can be implemented this PR adds a PDF viewer

## Screenshots (if appropriate):
phoenix-pdf-viewer

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by labkode** [comment]
@DeepDiver1975 the bottom menu to choose the application to handle the opening/editing of the file is just pure art! 🎨 :)

File: apps/pdf-viewer/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	plugins: [
+		new VueLoaderPlugin()
+	],
+	devtool: 'source-map',
+	entry: './src/app.js',
+	output: {
+		path: `${__dirname}/js`,
+		chunkFilename: '[name].pdf-viewer.chunk.js',
+		filename: "pdf-viewer.bundle.js",
+		publicPath: 'apps/pdf-viewer/js/'
```
Comment (scramb) [inline apps/pdf-viewer/webpack.config.js]: we should adjust this to fit the needs of the core webpack builder. publicPath should be 'apps/pdf-viewer/dist/' and path should be 'dist/'

**Comment by PVince81** [comment]
is it on purpose that the ownCloud header disappears ?

**Comment by DeepDiver1975** [comment]
> is it on purpose that the ownCloud header disappears ?

until we have #433 implemented there is only one fixed tool bar available. We will change this in the pdf viewer once we have this - end of this sprint ....

File: apps/pdf-viewer/webpack.config.js (hunk 1 above)
Comment (DeepDiver1975) [reply apps/pdf-viewer/webpack.config.js]: > we should adjust this to fit the needs of the core webpack builder. publicPath should be 'apps/pdf-viewer/dist/' and path should be 'dist/'

@scramb adjusted - please review again - THX

**Pull request merged**